src/lib/seam/components/DeviceDetails/NoiseSensorDeviceDetails.tsx

```diff
@@ -0,0 +1,149 @@
+import { ArrowRightIcon } from 'lib/icons/ArrowRight.js'
+import type { NestedSpecificDeviceDetailsProps } from 'lib/seam/components/DeviceDetails/DeviceDetails.js'
+import { DeviceInfo } from 'lib/seam/components/DeviceDetails/DeviceInfo.js'
+import { DeviceModel } from 'lib/seam/components/DeviceDetails/DeviceModel.js'
+import { useNoiseThresholds } from 'lib/seam/noise-sensors/use-noise-thresholds.js'
+import { DeviceImage } from 'lib/ui/device/DeviceImage.js'
+import { OnlineStatus } from 'lib/ui/device/OnlineStatus.js'
+import { DetailRow } from 'lib/ui/layout/DetailRow.js'
+import { DetailSection } from 'lib/ui/layout/DetailSection.js'
+import { DetailSectionGroup } from 'lib/ui/layout/DetailSectionGroup.js'
+
+interface NoiseSensorDeviceDetailsProps
+  extends NestedSpecificDeviceDetailsProps {
+  device: NoiseSensorDevice
+}
+
+export function NoiseSensorDeviceDetails({
+  device,
+  disableConnectedAccountInformation,
+  disableResourceIds,
+}: NoiseSensorDeviceDetailsProps): JSX.Element | null {
+  const { noiseThresholds, isInitialLoading } = useNoiseThresholds({
+    device_id: device.device_id,
+  })
+
+  return (
+    <div className='seam-device-details'>
+      <div className='seam-body'>
+        <div className='seam-summary'>
+          <div className='seam-content'>
+            <div className='seam-image'>
+              <DeviceImage device={device} />
+            </div>
+            <div className='seam-info'>
+              <span className='seam-label'>{t.noiseSensor}</span>
+              <h4 className='seam-device-name'>{device.properties.name}</h4>
+              <div className='seam-properties'>
+                <span className='seam-label'>{t.status}:</span>{' '}
+                <OnlineStatus device={device} />
+                <DeviceModel device={device} />
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <DetailSectionGroup>
+          <div className='seam-detail-section-wrap'>
+            <DetailSection
+              label='Noise thresholds'
+              tooltipContent={
+                device.device_type === 'minut_sensor' ? (
+                  <div className='seam-detail-section-tooltip-inner-content'>
+                    <span className='seam-tooltip-content'>
+                      A noise threshold is the highest noise level (in dB) you
+                      want to allow.
+                    </span>
+                    <span className='seam-tooltip-content'>
+                      Quiet hours is a separate threshold that takes effect only
+                      for a specified time range.
+                    </span>
+                  </div>
+                ) : (
+                  'A noise threshold is the highest noise level (in dB) you want to allow for a given time range in the day.'
+                )
+              }
+            >
+              {!isInitialLoading && (noiseThresholds?.length ?? 0) > 0 ? (
```

Comment:
Let's move all of the noise thresholds to a separate component so the loading logic, etc. is isolated.

Reply:
Due to the architecture of the UI, I had to move quite a lot, but it is now isolated!